tests/test_simple_pipeline_executor.py

```diff
@@ -243,14 +237,21 @@ def test_from_pipeline_output_differ(self):
             )
         )
 
+        executor = self._configure_pipeline(
```

Comment:
This has to move until after the dataset type has been registered? If that is the case shouldn't the other tests have this change?

Reply:
Good point. They probably don't matter, but they're at least unsafe.  Will do.